app/code/community/SomethingDigital/InfluxDb/Model/RequestProcessor.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+class SomethingDigital_InfluxDb_Model_RequestProcessor extends Enterprise_PageCache_Model_Processor
+{
+    const XML_PATH_SHOULD_TRACK = 'sd_influxdb/route_response_headers/enabled';
+
+    const XML_PATH_METADATA_SOURCE = 'sd_influxdb/route_response_headers/metadata_source';
```

Comment:
The won't pull from system.xml. They'll need to pull from an XML file in `app/etc` in order to work in the case of FPC.